packages/harmony/src/components/input/SelectablePill/SelectablePill.tsx

```diff
@@ -73,12 +96,34 @@ export const SelectablePill = (props: SelectablePillProps) => {
     }
   }
 
-  return (
-    <SelectablePillRoot {...props}>
+  const pillContent = (
+    <>
       {Icon ? <Icon css={iconCss} /> : null}
       <Text variant='body' tag='span'>
         {label}
       </Text>
-    </SelectablePillRoot>
+    </>
   )
+
+  if (type === 'checkbox' || type === 'radio') {
+    return (
+      <label css={[css, { position: 'relative' }]}>
```

Comment:
is there a reason not to put `position: relative` in the css above?

Reply:
it technically isnt needed for button, but good point might as well just combine